output/tar/tar_output.go

```diff
@@ -51,10 +51,9 @@ func (i Output) Apply(rootPath string) <-chan error {
 
 		err = tar.Run()
 		if err != nil {
-			done <- err
+			done <- output.Report{output.UnknownError.Wrap(err).(*errors.Error), def.Output{}}
 			return
 		}
-
 	}()
 	return done
```

Comment:
I see that a test got added for execing tar; are you comfortable with my Report thoughts above? We discussed out-of-band that an anon func for channel-returns might work well here; I'd be comfortable adapting each output implementation to 

1. Use that pattern
2. Always send a Report, regardless of success or fail

Let me know if you'd like to punt that to me, do it personally, or discuss this further.